ci-operator/config/kubevirt/hyperconverged-cluster-operator/kubevirt-hyperconverged-cluster-operator-master.yaml

```diff
@@ -34,6 +34,13 @@ images:
 - dockerfile_path: deploy/Dockerfile.registry.upgrade-prev
   from: operator-registry-base
   to: hco-registry-upgrade-prev
+operator:
+  bundles:
+  - context_dir: deploy/olm-catalog
+    dockerfile_path: bundle.Dockerfile
+  substitutions:
+  - pullspec: quay.io/kubevirt/hyperconverged-cluster-operator:1.2.0
```

Comment:
Please change to `quay.io/kubevirt/hyperconverged-cluster-operator@sha256:4df494494edb282bf0d781392a83cf42107d8d28be88822c8e6cb45e116425bc`, as a woraround, until we'll have a regex support.

Reply:
I'll get to this eventually. There are many more pullspecs in the CSV for which we'll likely need to define substitutions.

Reply:
This is the only one for the basic deployment test, but it's a must, because the current text (:1.2.0) is not in the csv anymore, but the one I wrote above.